frontend/src/components/account/sign-up/SignUp.test.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SignUp from "./SignUp";
+import { getDietaryRequirements } from "../../../utils/DietaryRequirements";
+
+test("renders 'sign up' page", () => {
+  const location = { state: undefined };
+  const { getByText } = render(<SignUp location={location} />);
+  const title = getByText("Sign Up");
+  expect(title).toBeInTheDocument();
+});
+
+test("renders 'change account details' page", () => {
```

Comment:
Hmm, but we're rendering the signup page on line 29.

Reply:
I use SignUp.js for both sign up and changing account details. The difference is that during 'sign up' the form is empty, but during 'change account details' the form is pre-filled with user details.

Reply:
ack, thanks.